sdk/communication/communication-call-automation/src/callConnection.ts

```diff
@@ -38,24 +38,24 @@ import { v4 as uuidv4 } from "uuid";
  */
 export class CallConnection {
   private readonly callConnectionId: string;
-  private readonly callConnectionImpl: CallConnectionImpl;
-  private readonly callMediaImpl: CallMediaImpl;
+  private readonly callConnection: CallConnectionImpl;
+  private readonly callMedia: CallMediaImpl;
 
   constructor(
```

Comment:
can we make this private constructor? 
CallConnection should be retrieved from GetCallConnection

Reply:
updated

Reply:
updated, and updated unit tests to match